### PR TITLE
Rowan test

### PR DESCRIPTION
**Pull request by rodoggyro**
Adds Vuforia and test OpModes

Adds TensorFlow and Vuforia test, autonomous testing program, and robot testing OpMode. Please read secrets.md file. Adds Vuforia api key required for code.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/secrets.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Handling FTC Secrets
+
+## What secrets are
+
+Secrets are things like api keys that you don't want to be made public. 
+
+## Creating the FTC Secrets Class
+
+When using a library like Vuforia where there is an api key you need to keep it hiden. This can be done with the following steps.
```
Comment (nshenoy) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/secrets.md:9]: typo: should be `hidden`

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TensorFlowTest.java [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+import org.firstinspires.ftc.robotcore.external.ClassFactory;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.VuforiaLocalizer;
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.robotcore.external.tfod.TFObjectDetector;
+
+import java.util.List;
+
+/**
```
Comment (nshenoy) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TensorFlowTest.java:43]: I think you were right when we were talking about this: a lof of these comments in the code that were in the original sample can probably be removed to make the *actual* code easier to see. Just make sure to keep relevant comments for sections of code to make it easier for the next person to understand. But the verbosity of the original comments can be cut down.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+@Autonomous
+
+public class TestAutonomous extends LinearOpMode{
+    DcMotor ramp;
+
+    public void runOpMode() {
+
+        //Setting up
+        double AOA = 15;
```
Comment (nshenoy) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java:16]: What does `AOA` stand for? Consider spelling it out - longer variable names are MUCH easier to read.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+
+@Autonomous
+
+public class TestAutonomous extends LinearOpMode{
+    DcMotor ramp;
```
Comment (nshenoy) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java:11]: Consider making this `rampMotor` to make it clear.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/secrets.md (hunk 1 above)
Comment (rodoggyro) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/secrets.md]: Thank you it's fixed now

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TensorFlowTest.java (hunk 2 above)
Comment (rodoggyro) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TensorFlowTest.java:43]: I will change that in our next push

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java (hunk 4 above)
Comment (rodoggyro) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java:11]: ok will do

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java (hunk 3 above)
Comment (rodoggyro) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestAutonomous.java:16]: It stands for Angle Of Attack or the angle the ramp is positioned

**Review by nshenoy — APPROVED** [review]

**Pull request merged**